types/qunit/index.d.ts

```diff
@@ -313,6 +314,36 @@ interface NestedHooks {
     
 }
 
+interface QUnitBeginDetails { totalTests: number }
```

Comment:
I don't think there is any reason to be prefixing these interfaces with QUnit.

Reply:
Better would be to make a `QUnit` namespace instead of adding multiple new globals.

Reply:
I'm open to renaming the interfaces, but keep in mind that these are types that often are **globally available**, and thus run the risk of colliding with other types in an project.

Reply:
At @andy-ms's suggestion -- I've moved the interfaces into a namespace